tools/omfwd.c

```diff
@@ -1140,7 +1140,7 @@ CODESTARTnewActInst
 	pvals = nvlstGetParams(lst, &actpblk, NULL);
```

Comment:
good catch! This actually looks like something else is not really correct in existing code... Will investigate.